js/card-popup.js

```diff
@@ -0,0 +1,80 @@
+import {getSimilarArray, SIMILAR_APARTMENTS_COUNT} from './data.js';
+
+const cardPopupTemplate = document.querySelector('#card').content.querySelector('.popup');
+const map = document.querySelector('.map__canvas');
+cardPopupTemplate.classList.remove('hidden');
+const TYPE_NAME = {
+  'flat': 'Квартира',
+  'bungalow': 'Бунгало',
+  'house': 'Дом',
+  'palace': 'Дворец',
+  'hotel': 'Отель',
+};
```

Comment:
константу в начало файла. 
И ещё, ключи не обязательно писать как строки, можно и так

```suggestion
const TYPE_NAME = {
  flat: 'Квартира',
  bungalow: 'Бунгало',
  house: 'Дом',
  palace: 'Дворец',
  hotel: 'Отель',
};
```